src/tests/JIT/interpreter/InterpreterTester.cs

```diff
@@ -16,15 +16,11 @@ public class InterpreterTester
     [Fact]
     public static void RunTests()
     {
-        string corerun = TestLibrary.Utilities.IsWindows ? "corerun.exe" : "corerun";
-        string libInterp = TestLibrary.Utilities.IsWindows ? "clrinterpreter.dll" : (TestLibrary.Utilities.IsMacOSX ? "libclrinterpreter.dylib" : "libclrinterpreter.so");
         string coreRoot = Environment.GetEnvironmentVariable("CORE_ROOT");
         string interpreterApp = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "Interpreter.dll");
 
-        var startInfo = new ProcessStartInfo(Path.Combine(coreRoot, corerun), interpreterApp);
-        startInfo.EnvironmentVariables["DOTNET_AltJitName"] = libInterp;
-        startInfo.EnvironmentVariables["DOTNET_AltJitPath"] = Path.Combine(coreRoot, libInterp);
-        startInfo.EnvironmentVariables["DOTNET_AltJit"] = "RunInterpreterTests";
+        var startInfo = new ProcessStartInfo(Path.Combine(coreRoot, "corerun"), interpreterApp);
```

Comment:
Is .exe not needed on windows?
```suggestion
        string corerun = TestLibrary.Utilities.IsWindows ? "corerun.exe" : "corerun";
        var startInfo = new ProcessStartInfo(Path.Combine(coreRoot, corerun), interpreterApp);
```

Reply:
No, it is not needed. It works both with and without it.

Reply:
It will at least keep things consistent with the existing practice; which may seem over-defensive or it's there to avoid unnecessary probes (CreateProcess will look for exact match, then fallback to appending .exe, .cmd .bat etc.)

```
src/libraries/System.Formats.Tar/tests/TarEntry/TarEntry.ExtractToFile.Tests.cs:            string linkTarget = PlatformDetection.IsWindows ? @"C:\Windows\system32\notepad.exe" : "/usr/bin/nano";
src/libraries/System.Formats.Tar/tests/TarEntry/TarEntry.ExtractToFileAsync.Tests.cs:                string linkTarget = PlatformDetection.IsWindows ? @"C:\Windows\system32\notepad.exe" : "/usr/bin/nano";
src/libraries/System.Formats.Tar/tests/TarFile/TarFile.ExtractToDirectory.Stream.Tests.cs:                entry.LinkName = PlatformDetection.IsWindows ? @"C:\Windows\System32\notepad.exe" : "/usr/bin/nano";
src/libraries/System.Formats.Tar/tests/TarFile/TarFile.ExtractToDirectoryAsync.Stream.Tests.cs:                    entry.LinkName = PlatformDetection.IsWindows ? @"C:\Windows\System32\notepad.exe" : "/usr/bin/nano";
src/tests/readytorun/coreroot_determinism/Program.cs:    public static string OSExeSuffix(string path) => (OperatingSystem.IsWindows() ? path + ".exe" : path);
src/tools/illink/test/ILLink.Tasks.Tests/ILLink.Tasks.Tests.cs:                 string dotnetToolName = OperatingSystem.IsWindows () ? "dotnet.exe" : "dotnet";
```